plugins/org.jkiss.dbeaver.ext.greenplum/src/org/jkiss/dbeaver/ext/greenplum/model/GreenplumSchema.java

```diff
@@ -91,9 +93,8 @@ public JDBCStatement prepareLookupStatement(@NotNull JDBCSession session,
                     "array_to_string(x." + execLocationColumn + ", ',') AS execlocation,\n" +
                     "pg_encoding_to_char(x.encoding) AS encoding,\n" +
                     "x.writable,\n")
-                    .append(container.getDataSource().isServerVersionAtLeast(9,4) ?
-                            "" :
-                            "case when x.fmterrtbl is not NULL then true else false end as \"is_logging_errors\",\n")
+                    .append(serverSupportFmterrtblColumn ?
+                            "case when x.fmterrtbl is not NULL then true else false end as \"is_logging_errors\",\n" : "")
```

Comment:
Can we use the same case as in previous parts of this query here? It looks inconsistent now.